src/jit/codegenxarch.cpp

```diff
@@ -4535,24 +4535,21 @@ void CodeGen::genRegCopy(GenTree* treeNode)
     assert(treeNode->OperGet() == GT_COPY);
     GenTree* op1 = treeNode->gtOp.gtOp1;
 
-    if (op1->IsMultiRegCall())
+    if (op1->IsMultiRegNode())
```

Comment:
Do you want to add a banner comment while you're here?

Reply:
Yes, good point. This method deserves a description.